contracts/token/Comptroller.sol

```diff
@@ -372,4 +372,12 @@ contract Comptroller is Controller, IComptroller {
             return nonMemberRatio;
         }
     }
+
+    /**
+     *  @dev Function to simply retrieve block timestamp
+     *  This exists mainly for inheriting test contracts to stub this result.
+     */
+    function getTimestamp() internal view returns (uint256) {
```

Comment:
Do we want to add this to the IComptroller and then add `override`?

Reply:
Why do you want to add an internal function to the interface?